DOC-1567: Updated the editor events documentation (#2314)
tinymce/tinymce-docs: modules/ROOT/pages/events.adoc

|PastePreProcess |`+{ content: string }+` |Fired before the content is parsed from the clipboard.
|PastePostProcess |`+{ node: Element }+` |Fired after the content has been parsed from the clipboard, but before it's added to the editor.
|PastePlainTextToggle |`+{ state: boolean }+` |Fired when the plain text paste mode state changes.
|AutocompleterStart |`+{ lookupData: AutocompleteLookupData[] }+` |Fired when an autocompleter is triggered and should show the fetched suggestions.

> |AutocompleterUpdate |`+{ lookupData: AutocompleteLookupData[] }+` |Fired when content is changed and the active autocompleter needs to be update the suggestions shown.

Reviewer: ```suggestion
|AutocompleterUpdate |`+{ lookupData: AutocompleteLookupData[] }+` |Fired when content is changed and the active autocompleter needs to update the suggestions shown.
```

> |AutocompleterEnd |N/A |Fired when the active autocompleter menu has been flagged to close.

Reviewer: This reads a bit weirdly to me. Mostly "flagged". Unsure what could be better though, so just mentioning it in case you can think of anything else. Feel free to ignore.

Author: Yeah I struggled with this as well and `flagged` was the best I could come up with as well sorry 😞

Author: ```suggestion
|AutocompleterEnd |N/A |Fired when the active autocompleter menu is about to be closed.
```
As discussed on slack, here's a somewhat better version at least.
